### DISCUSSION TITLE
[Bug] Unable to install java runtime environment

### DISCUSSION THREAD

**Discussion by yoke0104x** [1Panel-dev/1Panel#7565] [Q&A]
### Contact Information

yoke0104@163.com

### 1Panel Version

1.10.21

### Problem Description

无法安装java运行环境

### Steps to Reproduce

貌似是删除容器后，再删除运行环境，在创建就会有这个问题

### Related log output

```shell
无
```

### Additional Information

Uploading record.mp4…

**Comment by yoke0104x** [comment]
https://github.com/user-attachments/assets/9e7920c4-862a-4cba-85f7-1453870b4a4b

**Comment by zhengkunwang223** [comment]
你是飞牛os吗？现在我们还不支持，但是你可以去论坛找解决方案

**Comment by yoke0104x** [comment]
> 你是飞牛os吗？现在我们还不支持，但是你可以去论坛找解决方案

不是，就是liunx服务器

**Comment by zhengkunwang223** [comment]
> > 你是飞牛os吗？现在我们还不支持，但是你可以去论坛找解决方案
> 
> 
> 
> 不是，就是liunx服务器

docker-compose version 看一下版本

**Comment by yoke0104x** [comment]
> > > 你是飞牛os吗？现在我们还不支持，但是你可以去论坛找解决方案
> > 
> > 
> > 不是，就是liunx服务器
> 
> docker-compose version 看一下版本

[root@VM-20-7-opencloudos ~]# docker -v
Docker version 26.1.3, build b72abbb
[root@VM-20-7-opencloudos ~]# docker-compose -v
Docker Compose version v2.31.0
[root@VM-20-7-opencloudos ~]#

**Comment by zhengkunwang223** [comment]
> > > > 你是飞牛os吗？现在我们还不支持，但是你可以去论坛找解决方案
> > > 
> > > 
> > > 不是，就是liunx服务器
> > 
> > 
> > docker-compose version 看一下版本
> 
> [root@VM-20-7-opencloudos ~]# docker -v Docker version 26.1.3, build b72abbb [root@VM-20-7-opencloudos ~]# docker-compose -v Docker Compose version v2.31.0 [root@VM-20-7-opencloudos ~]#

看一下 /opt/1panel/runtime/java这个目录下有没有文件夹 有的话进去执行 docker-compose up 看一下

**Comment by wanghe-fit2cloud** [comment]
先将该 Issue 转为 Discusstion
